doc/users/whats_new.rst

```diff
@@ -131,7 +131,7 @@ Support for axes.prop_cycle property *markevery* in rcParams
 The Matplotlib ``rcParams`` settings object now supports configuration
 of the attribute `axes.prop_cycle` with cyclers using the `markevery`
 Line2D object property. An example of this feature is provided at
-`~/matplotlib/examples/lines_bars_and_markers/markevery_prop_cycle.py`
+:doc:`gallery/lines_bars_and_markers/markevery_prop_cycle`.
```

Comment:
```suggestion
:doc:`/gallery/lines_bars_and_markers/markevery_prop_cycle`.
```

At least that's what all other doc references are. (Can't check because of the broken doc build on CI and I'm too lazy to build the docs locally just for that :smile:).